code/script.js

```diff
@@ -1,19 +1,137 @@
-// Start here
-
 // Step 1 - Welcome and introduction
-// Your code goes here
 alert(
   `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
 )
 
+let name = prompt("What's your name?")
+if (name != null) {
+  alert(
+    `Hey there ${name}, are you ready to order?`
+    )
+  }
+else {
+  alert("Invalid choice, please try again!")
+  process.exit(1)
+}
+
 // Step 2 - Food choice
-// Your code goes here
+let menuChoices = "What type of food would you like to order? \n \n"
++ "\tEnter a number: \n"
++ "\t1. Pizza\n"
++ "\t2. Pasta\n"
++ "\t3. Salad \n"
```

Comment:
I don't really understand what the +  and the \t3. and the \n are are doing in this code, Im sure they have a reason to be there but as a newbie a comment with an explanation would have been useful. I also see that you have a space after salad on line 22 but no space on line 20 and 21, is there a reason for that?